src/sentry/incidents/logic.py

```diff
@@ -1849,7 +1841,7 @@ def is_included(action: Mapping[str, Any]) -> bool:
     ]
 
 
-def schedule_update_project_config(alert_rule: AlertRule, projects: Sequence[Project]):
+def schedule_update_project_config(alert_rule: AlertRule, projects: Collection[Project]) -> None:
```

Comment:
is `Collection` looser than `Sequence`?

Reply:
Yes. `Sequence` implies that it has a consistent iteration order and can be indexed with brackets, usually meaning a `list` or `tuple`. `Collection` includes `set` and `frozenset`.